src/test/rules/no-invalid-element-name_test.ts

```diff
@@ -78,8 +78,7 @@ ruleTester.run('no-invalid-element-name', rule, {
       customElements.define('my-App', MyApp)`,
       errors: [
         {
-          message:
-            'Custom element names must not contain uppercase ASCII characters.',
+          message: errorMessage,
```

Comment:
these can now be `messageId: 'invalidElementName'`, then pr looks good i think

Reply:
Cool, I've replaced the `message` for `messageId` in tests and also stop exporting it from the src file